platform.md

```diff
@@ -327,65 +376,57 @@ The lifecycle:
+| `<cache-dir>`  | `CNB_CACHE_DIR`       |                          | Path to a cache directory
+| `<cache-image>`| `CNB_CACHE_IMAGE`     |                          | Reference to a cache image in an OCI image registry
 | `<gid>`        | `CNB_GROUP_ID`        |                          | Primary GID of the stack `User`
 | `<group>`      | `CNB_GROUP_PATH`      | `<layers>/group.toml`    | Path to group definition (see [`group.toml`](#grouptoml-toml))
-| `<image>`      |                       |                          | Image reference to be analyzed (usually the result of the previous build)
 | `<layers>`     | `CNB_LAYERS_DIR`      | `/layers`                | Path to layers directory
 | `<log-level>`  | `CNB_LOG_LEVEL`       | `info`                   | Log Level
-| `<skip-layers>`| `CNB_SKIP_LAYERS`     | `false`                  | Do not perform layer analysis
 | `<uid>`        | `CNB_USER_ID`         |                          | UID of the stack `User`
-
-- **If** `<daemon>` is `false`, `<image>` MUST be a valid image reference
-- **If** `<daemon>` is `true`, `<image>` MUST be either a valid image reference or an imageID
-- The lifecycle MUST accept valid references to non-existent images without error.
+| `<skip-layers>`| `CNB_SKIP_LAYERS`     | `false`                  | Do not perform layer analysis
 
 ##### Outputs
-| Output             | Description
-|--------------------|----------------------------------------------
-| [exit status]      | (see Exit Code table below for values)
-| `/dev/stdout`      | Logs (info)
-| `/dev/stderr`      | Logs (warnings, errors)
-| `<analyzed>`       | Analysis metadata (see [`analyzed.toml`](#analyzedtoml-toml)
+| Output                                | Description
+|---------------------------------------|----------------------------------------------
+| [exit status]                         | (see Exit Code table below for values)
+| `/dev/stdout`                         | Logs (info)
+| `/dev/stderr`                         | Logs (warnings, errors)
 | `<layers>/<buidpack-id>/<layer>.sha`  | Files containing the diffID of each analyzed layer
 | `<layers>/<buidpack-id>/<layer>.toml` | Files containing the layer content metadata of each analyzed layer (see data format in [Buildpack Interface Specification](buildpack.md))
+| `<layers>/<buidpack-id>/<layer>/*`.   | Restored layer contents
 
 | Exit Code | Result|
 |-----------|-------|
 | `0`       | Success
 | `11`      | Platform API incompatibility error
 | `12`      | Buildpack API incompatibility error
 | `1-10`, `13-99` | Generic lifecycle errors
-| `200-299` | Analysis-specific lifecycle errors
+| `300-399` | Restoration-specific lifecycle errors
 
-- The lifecycle MUST write [analysis metadata](#analyzedtoml-toml) to `<analyzed>` if `<image>` is accessible.
 - **If** `<skip-layers>` is `true` the lifecycle MUST NOT perform layer analysis.
```

Comment:
```suggestion
- For each buildpack in `<group>`, if persistent metadata for that buildpack exists in the analysis metadata, lifecycle MUST write a toml representation of the persistent metadata to `<layers>/<buildpack-id>/store.toml`
- **If** `<skip-layers>` is `true` the lifecycle MUST NOT perform layer analysis.
```
We also need to modify the data format for `io.buildpacks.lifecycle.metadata`[https://github.com/buildpacks/spec/blob/main/platform.md#iobuildpackslifecyclemetadata-json] to included persistent metadata in the format currently expected by the lifecycle.

Reply:
What has changed here? Sorry, I don't see what needs to be modified.